### PR TITLE
Issue 46: Add role bindings with edit access

### PR DESCRIPTION
**Pull request by chetan-rns**
**What type of PR is this?**

> /kind feature

**What does this PR do / why we need it**:

- Converted `oc create rolebinding` command to role binding structs
- Updated role bindings to use edit access 
 
**Which issue(s) this PR fixes**:

Fixes https://issues.redhat.com/browse/GITOPS-46

### CODE REVIEW

File: pkg/pipelines/bootstrap.go [hunk 1]
```diff
@@ -1,196 +1,199 @@
...
-            Verbs:     []string{"create"},
-        },
-    }
+	dockerSecretName     = "regcred"
+	saName               = "pipeline"
+	roleName             = "tekton-triggers-openshift-demo"
+	roleBindingName      = "tekton-triggers-openshift-binding"
+	devRoleBindingName   = "demo-sa-admin-dev"
+	stageRoleBindingName = "demo-sa-admin-stage"
+	// PolicyRules to be bound to service account
```
Comment (wtam2018) [inline pkg/pipelines/bootstrap.go:28]: Needs better names.   "demo-sa" is gone.

File: pkg/pipelines/bootstrap.go [hunk 2]
```diff
@@ -1,196 +1,199 @@
...
+	out := make([]interface{}, 0)
+	sa := createServiceAccount(meta.NamespacedName(ns["cicd"], saName), dockerSecretName)
+	out = append(out, sa)
+	role := createRole(meta.NamespacedName(ns["cicd"], roleName), rules)
+	out = append(out, role)
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], roleBindingName), sa, role.Kind, role.Name))
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], "edit-clusterrole-binding"), sa, "ClusterRole", "edit"))
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], devRoleBindingName), sa, "ClusterRole", "edit"))
+	out = append(out, createRoleBinding(meta.NamespacedName(ns["cicd"], stageRoleBindingName), sa, "ClusterRole", "edit"))
+
```
Comment (wtam2018) [inline pkg/pipelines/bootstrap.go:131]: Need to create RoleBindings in "dev" and "stage" envirionments.

File: pkg/pipelines/bootstrap.go (hunk 1 above)
Comment (ishitasequeira) [reply pkg/pipelines/bootstrap.go:28]: made the mentioned changes

File: pkg/pipelines/bootstrap.go (hunk 2 above)
Comment (ishitasequeira) [reply pkg/pipelines/bootstrap.go:131]: made the mentioned changes

**Comment by wtam2018** [comment]
It looks like the whole file's space indentations have been replace by tabs.   Can you please fix it?

**Pull request merged**